tex/chProgramming.tex

```diff
@@ -555,7 +562,7 @@ \subsection{Local definitions}
 
 \begin{coq}{name=eval_let_expr}{width=6cm,title=Evaluating local definitions,label=lst:eval_let_expr}
 Eval compute in
-  let n := 33 in
+  let n := 33 : nat in
```

Comment:
I'm a bit uneasy with the type annotation begin used for `n` but not `e`. Why is that?